README.md

```diff
@@ -1 +1,69 @@
-# serverless-api
\ No newline at end of file
+# AWS API-Gateway, Dynamo and Lambda
+
+## Author: Eva Grace Smith
+
+### Task
+
+Create a serverless REST API.
+
+### Requirements
+
+Create a single resource REST API using a domain model, constructed using AWS Cloud Services.
+
+### Database: DynamoDB
+1 Table required.
+
+### Routing: API Gateway
+
+
+### UML
+
+![UML](./img/lab18-UML.png)
+
+### What is the root URL to your API?
+/people
+### What are the routes?
+
+Post:
+/people/handlePost
+
+Put:
+/people/handleUpdate
+
+Get:
+/people/handleRead
+
+Get by Id:
+/people/handleReadById
+
+Delete:
+/people/handleDelete
```

Comment:
The handler names are just the lambda functions that your API gateway is connecting to; a client knows nothing about "handlePost". Instead, your endpoints will be:

- POST: `/people`
- GET: `/people/{id}`
- etc.